parsons/action_kit/action_kit.py

```diff
@@ -637,9 +637,9 @@ def paginated_get_custom_limit(self, object_type, limit=None,
                 If "asc" (the default), return all objects below the threshold value.
                 If "desc", return all objects above the threshold value.
             **kwargs:
-                You can also add expressions to filter the data beyond the limit/threshold values above. For addition
-                info, visit `Django's docs on field lookups <https://docs.djangoproject.com/\
-                en/3.1/topics/db/queries/#field-lookups>`_.
+                You can also add expressions to filter the data beyond the limit/threshold values
+                above. For addition info, visit `Django's docs on field lookups
```

Comment:
If you wanted to take this opportunity to make a minor correction, you could change "addition info" to "additional info". (The original text had a minor typo.)

Reply:
Thanks! I updated it :)